### PR TITLE
Cleanup meta

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

**Comment by ef4** [comment]
Looks good.

File: packages/ember-metal/lib/property_events.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import {
-  guidFor
+  guidFor,
```
Comment (frankweb) [inline packages/ember-metal/lib/property_events.js]: Is this necessary?

Comment (frankweb) [inline packages/ember-metal/lib/property_events.js]: Do we need the comma?

**Comment by frankweb** [comment]
I also think this looks good.

**Comment by krisselden** [comment]
lgtm, this refactor will make future refactoring of meta much nicer to hide it behind an accessor.

File: packages/ember-metal/lib/property_events.js (hunk 1 above)
Comment (stefanpenner) [inline packages/ember-metal/lib/property_events.js]: oops, nope. Removing..

Comment (stefanpenner) [inline packages/ember-metal/lib/property_events.js]: https://github.com/emberjs/ember.js/pull/12455

**Pull request merged**